### PR TITLE
Feature/475-refactoring-qc-plot

### PR DESCRIPTION
**Pull request by PabloCorcho**
This PR includes a refactoring of the methods included in the qc_plot module, that now become class method of their respective target objects.

### CODE REVIEW

**Review by miguelglezb — COMMENTED** [review]
Besides the all caps variables mentioned in the review, all good!

File: src/pykoala/plotting/utils.py [hunk 1]
```diff
@@ -65,13 +75,8 @@ def new_figure(fig_name,
...
     return fig, axes
 
-
-default_cmap = plt.get_cmap("gist_earth").copy()
-default_cmap.set_bad('gray')
-
-
 def colour_map(fig, ax, cblabel, data,
-               cmap=default_cmap,
+               cmap=DEFAULT_CMAP,
```
[the remark this answers is not in the document]
Comment (PabloCorcho) [reply src/pykoala/plotting/utils.py:79]: That is exactly why I changed it

File: src/pykoala/plotting/utils.py [hunk 2]
```diff
@@ -1,13 +1,23 @@
 from matplotlib import pyplot as plt
 from matplotlib import colors
 from matplotlib.ticker import AutoMinorLocator
+from matplotlib.collections import PatchCollection
+from matplotlib.cm import ScalarMappable
 
 import numpy as np
 
+from astropy import units as u
```
[the remark this answers is not in the document]
Comment (paranoya) [reply src/pykoala/plotting/utils.py:9]: I guess it is an informal convention, such as `import numpy as np`

File: src/pykoala/plotting/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (paranoya) [reply src/pykoala/plotting/utils.py:79]: Is `THROUGHPUT_CMAP` a constant too? Why is it defined here?

**Review by paranoya — APPROVED** [review]

File: src/pykoala/plotting/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (miguelglezb) [reply src/pykoala/plotting/utils.py:9]: All good then.

File: src/pykoala/plotting/utils.py [hunk 3]
```diff
@@ -151,12 +156,12 @@ def colour_map(fig, ax, cblabel, data,
 
     return im, cb
 
-def fibre_map(fig, ax, cblabel, data, rss=None, fib_ra=None, fib_dec=None,
-              s=100, cmap=default_cmap, norm=None, cbax=None,
-              norm_interval=AsymmetricPercentileInterval,
-              interval_args={"lower_percentile": 1.0,
-                              "upper_percentile": 99.0},
-              stretch=PowerStretch, stretch_args={"a": 0.7}):
+def plot_fibres(fig, ax, rss=None, x=None, y=None,
```
Comment (miguelglezb) [inline src/pykoala/plotting/utils.py:159]: Originally, in tutorial 1, `fibre_map` uses parameter `fib_ra`, which is not present in `plot_fibres`. I suggest whether adding `fib_ra=None` in `plot_fibres` or remove the parameter in the tutorial.

**Review by miguelglezb — APPROVED** [review]

File: src/pykoala/plotting/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (PabloCorcho) [reply src/pykoala/plotting/utils.py:87]: That variable has been replaced by a generic default colormap

**Pull request merged**